nci/lib/watcher.rb

```diff
@@ -260,10 +270,22 @@ def run
         #end
       #end
 
-      bump_version
-
-      cmd.run('git --no-pager diff')
-      cmd.run("git commit -a -vv -m 'New release'")
+      # check if future_is_early and make sure correct series is applied and
+      # bump both release_* branches if required
+      if NCI.future_is_early == 'false'
```

Comment:
I may be mistaken but I think future_is_early is a literal bool type, not a string.